src/app/pages/receive/receive-collectible/receive-collectible.layout.tsx

```diff
@@ -0,0 +1,84 @@
+import toast from 'react-hot-toast';
+import { FiCopy } from 'react-icons/fi';
+import { useNavigate } from 'react-router-dom';
+
+import { Box, Button, Flex, Stack, useClipboard } from '@stacks/ui';
+import { truncateMiddle } from '@stacks/ui-utils';
+
+import { RouteUrls } from '@shared/route-urls';
+
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { StxAvatar } from '@app/components/crypto-assets/stacks/components/stx-avatar';
+import { OrdinalIcon } from '@app/components/icons/ordinal-icon';
+import { Flag } from '@app/components/layout/flag';
+import { Caption } from '@app/components/typography';
+import { useNextFreshTaprootAddressQuery } from '@app/query/bitcoin/ordinals/use-next-fresh-taproot-address.query';
+import { useCurrentAccountStxAddressState } from '@app/store/accounts/blockchain/stacks/stacks-account.hooks';
+
+export function ReceiveCollectibleLayout() {
+  const analytics = useAnalytics();
+  const navigate = useNavigate();
+  const { isLoading, isError, data: btcAddress } = useNextFreshTaprootAddressQuery();
+
+  const stxAddress = useCurrentAccountStxAddressState();
```

Comment:
If this is a layout component, it should be responsible for the layout of the component only, and these values passed in as props from the parent.